examples/howto/server_embed/flask_gunicorn_embed.py

```diff
@@ -1,7 +1,7 @@
 try:
     import asyncio
 except ImportError:
-    raise RuntimeError("This example requries Python3 / asyncio")
+    raise RuntimeError("This example requires Python3 / asyncio")
```

Comment:
I think this `try/catch` could actually be removed entirely? Bokeh itself requires Python 3 nowadays, so `asyncio` will always be available.

Reply:
Okay got it, implemented in a43a152.